recipes/annosine2/meta.yaml

```
@@ -0,0 +1,67 @@
+{% set name = "annosine2" %}
+{% set version = "1.0.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c9d12adec9875c4a429c6e9da5b76d31c760ef7474635caf742d25592378f34c
+
+build:
+  number: 0
+  entry_points:
+    - annosine2 = AnnoSINE.AnnoSINE_v2:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+  run_exports:
+        - {{ pin_subpackage('annosine2', max_pin="x") }}
+  noarch: python
+
+requirements:
+  host:
+    - cycler ==0.10.0
+    - kiwisolver
+    - matplotlib-base
+    - numpy
+    - pillow
+    - pip
+    - pyparsing
+    - python
+    - python-dateutil 
+    - six
```

```suggestion
    - pip
    - python
```
Are all these dependencies required during build time?

---

Oh, I think only pip and python are required during build time. Thanks for the change and review!